core/services/request/sanitizers/MultipleAttributes.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace EventEspresso\core\services\request\sanitizers;
+
+/**
+ * Class MultipleAttributes
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\services\request\sanitizers
+ * @since   4.10.30
+ */
+class MultipleAttributes
```

Comment:
can you plz rename this class to `AttributesSanitizer` and update the file name accordingly?

Reply:
Since it's already part of Sanitizers namespace, I thought that `MultipleAttributes` is cleaner and present the function of this class better but will do that.

Reply:
Yes, great points. Renamed the class 👍